plugins/inputs/sqlserver/azuresqlqueries.go

```diff
@@ -449,13 +449,19 @@ WITH PerfCounters AS (
 			,'Readahead pages/sec'
 			,'Lazy writes/sec'
 			,'Checkpoint pages/sec'
+			,'Free pages'
+			,'Extension free pages'
```

Comment:
Is this really something we want? Very few folks ever use BP extensions

Reply:
This was based on a request. However, since these changes have been pulled out of this PR, I will address this offline or in the next PR.